packages/react-composites/src/composites/CallComposite/utils/Utils.ts

```diff
@@ -459,7 +460,6 @@ export const getBackgroundEffectFromSelectedEffect = (
     ? new BackgroundReplacementEffect({ backgroundImageUrl: selectedEffect.backgroundImageUrl })
     : undefined;
 
-/* @conditional-compile-remove(video-background-effects) */
```

Comment:
Why did this get removed?

Reply:
The function used for this pr is conditionally compiled under video-background-effects, so need to remove it to make it work in stable